### PR TITLE
packages correctly loaded for future_lapply

### PR DESCRIPTION
**Pull request by jeremyrcoyle**
`future_lapply` behaves differently for `plan(batchtools_custom)` then for, e.g., `plan(session)` because the former doesn't use a packages argument. This small PR makes the behavior for `future.batchtools` plans more closely match the plans from the `future` package.

*BatchtoolsFuture-class.R now uses its packages argument.
*tests/packages.R is a MWE to verify this

### CODE REVIEW

**Comment by jeremyrcoyle** [comment]
MWE has unmet dependencies. Let me try again.

**Comment by HenrikBengtsson** [comment]
Thanks. Otherwise I'll look into it because this is on my todo list of updates.
